build/action.yaml

```diff
@@ -35,18 +40,49 @@ runs:
       uses: andstor/file-existence-action@v3
       with:
         files: yarn.lock
+    - name: Compute lockfile hash
+      if: inputs.s3-bucket-name != ''
+      id: lockfile_hash
+      run: echo "hash=${{ steps.check_yarn_lock.outputs.files_exists == 'true' && hashFiles('**/yarn.lock') || hashFiles('**/package-lock.json') }}" >> $GITHUB_OUTPUT
```

Comment:
computing the hash from the yarn.lock or package-lock.json file depending on what installer we're using